src/50_Samples_%26_Templates/Product.html

```diff
@@ -513,7 +513,8 @@ <h2 class="m1">Apple</h2>
           <amp-selector name="color"
                   layout="container"
                   [selected]="selectedColor"
-                  on="select:AMP.setState(selectedColor=event.targetOption, selectedSlide=0, selectedSize='S')">
+                  on="select:AMP.setState({selectedColor:event.targetOption, selectedSlide:0,
+                    selectedSize:product[event.targetOption].sizes[selectedSize] != null ? selectedSize : product[event.targetOption].defaultSize})">
```

Comment:
* Does it work without `!= null` in `product[event.targetOption].sizes[selectedSize] != null`. It would in JS.